spicepy/_client.py

```diff
@@ -71,5 +78,8 @@ def query(self, query: str) -> flight.FlightStreamReader:
             reader = self._flight_client.do_get(
                 flight_info.endpoints[0].ticket, self._flight_options
             )
+        except flight.FlightTimedOutError as exc:
+            self._flight_client.cancel(flight_info.endpoints[0].ticket, self._flight_options)
+            raise TimeoutError(f"Flight request timed out: {DEFAULT_QUERY_TIMEOUT_SECS}") from exc
```

Comment:
```suggestion
            raise TimeoutError(f"Flight request timed out: {DEFAULT_QUERY_TIMEOUT_SECS}") from exc
```

This will always show the default timeout even if they have specified a custom timeout.

Reply:
Additionally, it just shows a number. It should say something like `Query timed out and was canceled after {timeout} seconds.`

Reply:
message updated